src/utils/db.ts

```diff
@@ -1,12 +1,3 @@
-import {Sequelize} from 'sequelize-typescript';
-import path from 'path';
-import {DATA_DIR} from '../services/config.js';
-import {FileCache, KeyValueCache, Settings, Shortcut} from '../models/index.js';
+import PrismaModule from '@prisma/client';
```

Comment:
```suggestion
import {PrismaClient} from '@prisma/client';
```

Reply:
<img width="1546" alt="Screen Shot 2565-01-03 at 19 48 18" src="https://user-images.githubusercontent.com/10498040/147932390-ef5bcbf3-ed4e-473f-8a1c-b5c44e89f68b.png">

I got this error when import like that
any suggestions?

Reply:
Looks like Prisma is only publishing a CommonJS module for now, so we'll leave it as-is.